rs/messaging/src/routing/stream_handler/tests.rs

```diff
@@ -1998,12 +1998,16 @@ fn induct_stream_slices_partial_success() {
                 // ...a request to a missing canister @46 (on this subnet according to the
                 // routing table); this is expected to trigger a reject response...
                 Request(*REMOTE_CANISTER, *OTHER_LOCAL_CANISTER),
-                // ..a request not from `REMOTE_SUBNET` @47...
+                // ...a request not from `REMOTE_SUBNET` @47; this is also expected to trigger
+                // a reject respose...
                 Request(*LOCAL_CANISTER, *LOCAL_CANISTER),
-                // ...a request from a missing canister @48 (not anywhere according to the routing
-                // table); this is expected to be accepted but dropped...
+                // ...responses not from `REMOTE_SUBNET` @48 and @49; this is expected to be dropped...
+                Response(*LOCAL_CANISTER, *LOCAL_CANISTER),
+                Response(*UNKNOWN_CANISTER, *LOCAL_CANISTER),
+                // ...a request from a missing canister @50 (not anywhere according to the routing
+                // table); this is expected to be a reject response...
```

Comment:
We also use "missing canister" below, to refer to a canister that's mapped to this subnet by the routing table but does not exist. This has a completely different meaning.

```suggestion
                // ...a request from a canister @50 not hosted anywhere according to the routing
                // table; expected to produce a reject signal...
```